frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -630,58 +631,58 @@ export default ({
         console.error(`Error while editing message(${message.hash}) in chatroom(${contractID})`, e)
       })
     },
-    deleteMessage (message) {
-      if (!this.isMsgSender(message.from)) {
-        alert('TODO: Coming soon...')
-        return
-      }
+    async deleteMessage (message) {
       const contractID = this.renderingChatRoomId
       const manifestCids = (message.attachments || []).map(attachment => attachment.downloadData.manifestCid)
-      sbp('gi.actions/chatroom/deleteMessage', {
-        contractID,
-        data: { hash: message.hash, manifestCids, messageSender: message.from }
-      }).catch((e) => {
-        console.error(`Error while deleting message(${message.hash}) for chatroom(${contractID})`, e)
-      })
+
+      const originalText = message.text || (message.attachments?.[0].name || '')
+      const text = convertToMarkdown(swapUserIDForUsername(originalText))
+      const promptConfig = {
+        heading: L('Delete message'),
+        question: L('Are you sure you want to delete this message permanently?{textPreview}', {
+          textPreview: `<p><span class="custom-markdown-content">${text}</span></p>`
+        }),
+        primaryButton: L('Yes'),
+        secondaryButton: L('Cancel')
+      }
```

Comment:
I have a few requests here:

1. I don't think we need to show a re-rendering of the message here. We can simply ask if they're sure if they want to delete the message permanently.
2. I think the function `convertToMarkdown` is named in a rather confusing way. This is something I missed from my previous PR review of @SebinSong's PR. It sounds like it's taking HTML and converting it to Markdown. But that's not what it's doing. Instead, it's taking Markdown and converting it to HTML (the opposite). So could you please refactor/rename this function to something like, `renderMarkdown`?
3. Please use two different strings here, depending on the situation:
  - No attachments: `Are you sure you want to delete this message permanently?`
  - Attachments: `Are you sure you want to delete this message and its file attachments permanently?`

Reply:
Okay, Will do.